README.md

```diff
@@ -7,7 +7,7 @@ Figma UI doc: https://www.figma.com/design/NdSqaZV0ZHHL3NxXy5vjDd/Figma-practice
 
```

Comment:
I spend 50 mins to finish this commit: 
Learnings: 
- https://chatgpt.com/share/671ca0dc-6f60-800a-8246-e77acc0b5cee
- https://chatgpt.com/share/671ca0ec-0fe0-800a-bd19-aaf01417249b